src/RadiativeTransfer/RadiativeTransfer.jl

```diff
@@ -1,19 +1,29 @@
 module RadiativeTransfer
 using ...Korg: PlanarAtmosphere, ShellAtmosphere, get_tau_5000s, get_zs
-
-# for generate_mu_grid
 using FastGaussQuadrature: gausslegendre
+
 """
     generate_mu_grid(n_points)
+    generate_mu_grid(μ_values)
 
 Used by both radiative transfer schemes to compute quadature over μ. Returns `(μ_grid, μ_weights)`.
+If an integer is passed, generate a grid of Gauss-Legendre quadrature points of corresponding size.
+Otherwise, use the provided μ values.
 """
-function generate_mu_grid(n_points)
+function generate_mu_grid(n_points::Integer)
     μ_grid, μ_weights = gausslegendre(n_points)
     μ_grid = @. μ_grid/2 + 0.5
     μ_weights ./= 2
     μ_grid, μ_weights
 end
+function generate_mu_grid(μ_grid::AbstractVector{<:Real})
```

Comment:
Out of curiosity, is a "range" handled by this case? (it probably doesn't matter)

Reply:
Yes!

Reply:
Neat!